sample/build.gradle

```diff
@@ -23,3 +23,13 @@ dependencies {
 
   testImplementation libs.testParameterInjector
 }
+
+tasks.withType(Test).configureEach {
+  testLogging {
+    events 'passed', 'failed', 'skipped', 'standardOut', 'standardError'
+    exceptionFormat 'full'
+    showExceptions true
+    showStackTraces true
+    showCauses true
+  }
+}
```

Comment:
mind updating this instead? https://github.com/cashapp/paparazzi/blob/master/build.gradle#L30